roofit/roofitcore/src/RooRealSumPdf.cxx

```diff
@@ -44,30 +44,22 @@ to the fractions of the various functions. **This requires setting the last argu
 
 */
 
-#include "RooFit.h"
-#include "Riostream.h"
-
-#include "TError.h"
-#include "TIterator.h"
-#include "TList.h"
-#include "TClass.h"
 #include "RooRealSumPdf.h"
+
+#include "RooRealIntegral.h"
 #include "RooRealProxy.h"
-#include "RooPlot.h"
 #include "RooRealVar.h"
-#include "RooAddGenContext.h"
-#include "RooRealConstant.h"
-#include "RooRealIntegral.h"
 #include "RooMsgService.h"
-#include "RooNameReg.h"
+
+#include <TError.h>
 
 #include <algorithm>
 #include <memory>
+#include <exception>
```

Comment:
did you mean `<stdexcept>`?

Reply:
Noooo, I wouldn't make that mistake?!? :sweat_smile: